README.md

```diff
@@ -12,22 +12,37 @@
 
 </div>
 
-`torchtitan` is currently in a pre-release state and under extensive development. Currently we showcase pre-training **Llama 3.1** LLMs of various sizes from scratch. To use the latest features of `torchtitan`, we recommend using the most recent PyTorch nightly.
+`torchtitan` is currently in a pre-release state and under extensive development. Currently we showcase training Llama 3.1 LLMs at scale. We are working on other types of generative AI models, including LLMs with MoE architectures, multimodal LLMs, and diffusion models, in the [`experiments`](torchtitan/experiments) folder.
```

Comment:
nit: start the second sentence with 'We' since currently is already in the first sentence.